pull_request_package/lib/github_status_reporter.rb

```diff
@@ -3,22 +3,57 @@
 
 class GitHubStatusReporter
   include ActiveModel::Model
-  attr_accessor :client, :logger, :pull_request
+  attr_accessor :client, :logger, :package
   
   def report
-    client.create_status('openSUSE/open-build-service', 'f73658fa927b90a0a3d28b79866215210b856390', 'pending', options) if [:success, :failed].include?(state)
+    summary = self.summary
```

Comment:
If you use an instance variable like I mentioned in my other comment, you can get rid of this variable completely.